quizgen/latex.py

```diff
@@ -37,10 +86,14 @@ def set_cli_args(parser):
                 + ' If not specified, $PATH will be searched.'
                 + ' Used to compile PDFs.'))
 
+    parser.add_argument('--use-docker', dest = 'use_docker',
```

Comment:
Scope this flag specifically to pdflatex.
Note that this function is used to modify an already existing parser,
we don't want to accidentally clobber an existing `--use-docker` flag.
So, something like `--pdflatex-use-docker` instead.